src/shapes/Path.ts

```diff
@@ -36,12 +37,14 @@ export class Path extends FabricObject {
 
   /**
    * Constructor
-   * @param {Array|String} path Path data (sequence of coordinates and corresponding "command" tokens)
+   * @param {TSimplePathData} path Path data (sequence of coordinates and corresponding "command" tokens)
+   * @param {Number} left
+   * @param {Number} top
```

Comment:
mostly sure this is incorrect as JSDOC.

Reply:
You're right, fixed

Reply:
Maybe I'll go through and replace all the incorrect elementary typed jsdoc comments like this in another PR, because they're a lot of them everywhere

Reply:
just remove this 2 new lines you added line 41 and 42